share/ansible_navigator/utils/catalog_collections.py

```diff
@@ -29,8 +29,10 @@
 except ImportError:
     from yaml import SafeLoader  # type: ignore
 
+# pylint: disable=import-error
 from key_value_store import KeyValueStore  # type: ignore
 
+# pylint: enable=import-error
```

Comment:
Try to always disable things inline
```suggestion
from key_value_store import KeyValueStore  # type: ignore  # pylint: disable=import-error

```

Reply:
cool, ty

Reply:
A bit more context: comments (any comments) tend to float around the code base. If you add a comment on top of a line, there's no guarantee that in the future, an unrelated line won't appear between a comment and its context. Putting comments on the same line has a better chance of surviving such situation and you won't see cases of misleading comments explaining something that doesn't seem related to the code below (if one line can sneak in, a block of code can too, just give it time).

This is why my rule of thumb is to move the pylint comment above the line only if there are technical limitations that prevent inlining, like linters complaining about the line being too long.